docs/source/tutorials/psij.ipynb

```diff
@@ -178,7 +170,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The [wait](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job.wait) call will return once the job has reached a terminal state, which almost always means that it finished or was cancelled.\n",
+    "The [wait](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job.wait) call will return once the job has reached a terminal state, which means either completed, failed, or cancelled.\n",
     "\n",
     "To distinguish jobs that complete successfully from ones that fail or are cancelled, fetch the status of the job after calling [wait](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job.wait):"
```

Comment:
Might want to also change this sentence to reflect that you are using the status returned from `wait`.